lib/ipmi_backend_utils.pm

```diff
@@ -145,8 +146,27 @@ sub umount_installation_disk {
     assert_script_run("ls $mount_point");
 }
 
+#Get the partition where the new installed system is installed to
 sub get_installation_partition {
-    my $partition = script_output("fdisk -l | grep \"^\/dev\/sda\.\*\\\*\" | cut -d ' ' -f 1");
+    my $partition = '';
+
+    #Do not use script_output because when the command fail, script_output dies
+    type_string(qq{fdisk -l | grep "^\/dev\/sda\.\*\\\*" | cut -d ' ' -f 1 | tee /dev/$serialdev\n});
+    $partition = wait_serial;
+    save_screenshot;
+    if (is_storage_ng && ($partition eq '')) {
+        record_soft_failure "bsc#1080729 - Partitioner does not mark boot flag";
+        my $y2log_file                = '/var/log/YaST2/y2log';
+        my $root_partition_commit_msg = script_output(qq{grep 'Commit Action "Adding mount point / of .* to /etc/fstab' $y2log_file});
+        $root_partition_commit_msg =~ m#Commit Action \"Adding mount point / of ([^\s]*) to /etc/fstab#m;
```

Comment:
I think this was change partially. The escapings are still there. And I assume okurz will complaint about using the character '#'. Did I write '#' in the suggestion?

```perl
$root_partition_commit_msg =~ m{Commit Action "Adding mount point / of ([^\s]*) to /etc/fstab}m;
# or
$root_partition_commit_msg =~ m{Commit Action "Adding mount point / of (\S*) to /etc/fstab}m;
```